content/embeddable-view/embeddable-view-settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "host": "https://bafybeidfltufjabou4obph2yoahk2cva3gjaritqy3djrkpavbelyygjyu.on.fleek.co/",
```

Comment:
@amanda-agency-undone @svvimming Are these settings being added to `settings.json` instead of this dedicated file?

Reply:
@orvn There were no plans to do so, but it sounds like a good idea. The embeddable view will need to be refactored a bit.

Reply:
@amanda-agency-undone Let's do it